cloudconnectlib/core/http.py

```diff
@@ -221,27 +221,15 @@ def _send_internal(self, uri, method, headers=None, body=None, proxy_info=None):
                 verify=self.requests_verify,
             )
         except SSLHandshakeError:
-            _logger.warning(
+            _logger.error(
```

Comment:
Post this change, do we want to return anything or raise another exception that needs to be caught and addressed by the TA using this?

Reply:
Once the __send_internal()_ function will encounter an exception then it will return nothing and then in the __retry_send_request_if_needed()_ function, it will encounter an another exception `AttributeError: 'NoneType' object has no attribute 'content'` and hence the execution will be terminated, which is what we want to achieve.

Reply:
In that case, we can raise the exception and let the TAs using this library handle it on their own. Breaking the code flow by raising an exception would be better than returning nothing.

Reply:
Done - https://github.com/splunk/addonfactory-cloudconnect-library/pull/144/commits/3cdcfb60a392321180de9a12fda2dee9c6375610